formats/financial_releases_campaign/publisher/details.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "image",
+     "body"
+   ],
+  "properties": {
+     "image": {
+       "type": "object",
+       "additionalProperties": false,
+       "properties": {
+         "url": {
+           "type": "string",
+           "format": "uri"
+         },
+         "alt_text": {
+           "type": "string"
+         }
+       }
+     },
+    "header_legal_disclaimer": {
```

Comment:
It would be great if we could add small descriptions for these fields if they're not immediately obvious.